packages/jest-editor-support/src/index.js

```diff
@@ -15,7 +15,7 @@ const ProjectWorkspace = require('./ProjectWorkspace');
 const Runner = require('./Runner');
 const Settings = require('./Settings');
 const {Expect, ItBlock, Node} = require('./parsers/ParserNodes');
-const {parse} = require('./ScriptParser');
+const {babylonParser: parse} = require('./parsers/BabylonParser');
```

Comment:
can we rename the thing it exports to `parse` so we don't have to alias it? It makes sense to me to call it `BabylonParser.parse`.